docs/installation.rst

```diff
@@ -0,0 +1,33 @@
+************
+Installation
+************
+
+Requirements
+============
+
+**astroplan** requires Python 2.7 or 3.3+ (2.6 and 3.2 or earlier are not
+supported) as well as the following packages:
+
+* `Numpy <http://www.numpy.org>`_
+* `Matplotlib <http://www.matplotlib.org>`_
+* `Astropy <http://www.astropy.org>`_
```

Comment:
There's this Sphinx warning that makes the travis-ci build fail:
https://travis-ci.org/astroplanners/astroplan/jobs/71459794#L893

It points to a line number that doesn't exist ... oh my.
You'll have to debug this locally by running `python setup.py build_sphinx -l` locally until you see no more warnings.
I think the culprit might be this line, and you can fix it by adding two underscores at the end, i.e.
```
`Astropy <http://www.astropy.org>`__
```